app/src/main/java/com/doublesp/coherence/actions/ListFragmentActionHandler.java

```diff
@@ -97,6 +90,9 @@ public void afterTextChanged(Editable s) {
         if (s.subSequence(i - 1, i).toString().equals("\n")) {
             mIdeaInteractor.addIdea(s.toString().trim());
             s.clear();
+            List<Idea> ideas = mIdeaInteractor.getPlan().getIdeas();
+            mShoppingListDatabaseReference.child(mIdeaInteractor.getPlan().getId()).child(
+                    ConstantsAndUtils.IDEAS).setValue(ideas);
```

Comment:
so it is pending on #75, I'll implement a row-based state change so that we could store the individual idea.